src/System.Management.Automation/security/Authenticode.cs

```diff
@@ -292,7 +292,7 @@ internal static Signature GetSignature(string fileName, string fileContent)
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Reliability", "CA2001:AvoidCallingProblematicMethods")]
         private static Signature GetSignatureFromCatalog(string filename)
         {
-            if (Signature.CatalogApiAvailable.HasValue && !Signature.CatalogApiAvailable.Value)
+            if (Signature.CatalogApiAvailable.HasValue && !Signature.CatalogApiAvailable.GetValueOrDefault())
```

Comment:
@iSazonov Why changing to `GetValueOrDefault()`? `HasValue` is already checked and thus it's fine to access `.Value`, not?

Reply:
@daxian-dbw It is a recommendation from .Net team https://github.com/dotnet/runtime/issues/33792 I opened #13791 for Hacktoberfest exclusively. It is not critical for us (no hot paths) but I hoped it is good for first time contributions.

Reply:
I think the comment https://github.com/dotnet/runtime/issues/33792#issuecomment-649665286 in that issue has a point. `.Value` is semantically more correct given that `HasValue` is already checked. There is a duplicate check on the boolean `hasvalue` field, but I don't think that affects the perf in practice.

For places where we can use `GetValueOrDefault` to replace both `.HasValue` and `.Value`, it makes more sense to make the change, like in #13805 and #13808. But if we need to keep the `.HasValue` check, then I think using `.Value` afterwards is better.

Reply:
I agree in common. Only there is another side - using different patterns decreases readability. I would prefer to use one pattern even though it looks unusual, especially since there are only a few of them in our code base. There is another reason why I prefer this. Now we often look at a code in .Net Runtime for a better understanding of how an application works and we consider this code as the best practice and high quality. Switching to a different code with a different style immediately causes discomfort. I believe that following .Net in this sense is the right direction - many of those who learn PowerShell code will be grateful to us.